llvm/test/Bitcode/thinlto-alias.ll

```diff
@@ -21,7 +21,7 @@
 ; CHECK-NEXT:    <FLAGS
 ; See if the call to func is registered.
 ; The value id 1 matches the second FUNCTION record above.
-; CHECK-NEXT:    <PERMODULE {{.*}} op7=1/>
+; CHECK-NEXT:    <PERMODULE_PROFILE {{.*}} op7=1
```

Comment:
Can you split out the renaming of s/PERMODULE/PERMODULE_PROFILE and s/COMBINED/COMBINED_PROFILE as an NFC?

Then it would be clear exactly which expectations changed.

Reply:
I considered that but it isn't just a straight NFC renaming - we have to change the format at the use sites too. I.e. we would have to make all the previously non-PROFILE cases in the BitcodeWriter create a PROFILE version with just an unknown hotness value. Which is doable but I wasn't sure if it was worth it since it doesn't make sense on its own. wdyt?

Reply:
Ok, sounds like too much toil to split them apart.